app/styles/front.scss

```diff
@@ -347,17 +347,7 @@ $autre-lighter: #cce4e7;
 
```

Comment:
Maybe this is more a question to discuss in real life, but what are the good practises in term of css files ? 
I feel rather lost when working with this unique big file, is there smart way to split the CSS ?